### PR TITLE
Update Contributing to install PG10 instead of 9.6

### PR DESCRIPTION
**Pull request by sumedhpathak**

### CODE REVIEW

**Comment by jasonmp85** [comment]
Merged just in time for the upcoming PG11 release :-P

**Comment by sumedhpathak** [comment]
:) I debated PG11, but wanted something I can test out the instructions.

**Comment by sumedhpathak** [comment]
@jasonmp85 Does the change look good to merge?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ why we ask this as well as instructions for how to proceed, see the
 
 #### Red Hat-based Linux (RHEL, CentOS, Fedora)
 
-1. Find the PostgreSQL 9.6 RPM URL for your repo at [yum.postgresql.org](http://yum.postgresql.org/repopackages.php#pg96)
+1. Find the PostgreSQL 10 RPM URL for your repo at [yum.postgresql.org](http://yum.postgresql.org/repopackages.php#pg96)
```
Comment (jasonmp85) [inline CONTRIBUTING.md]: This should link to the pg10 section.

```suggestion
1. Find the PostgreSQL 10 RPM URL for your repo at [yum.postgresql.org](http://yum.postgresql.org/repopackages.php#pg10)
```

**Comment by sumedhpathak** [comment]
Ooh, fancy. I like the suggested feature. Applied your change. Will ship it after the tests pass. Thanks!

**Comment by sumedhpathak** [comment]
Also, maybe you want to approve after tests pass? I wouldn't want to dismiss your review.

**Comment by jasonmp85** [comment]
> Also, maybe you want to approve after tests pass?

Approved now; there's no point in waiting for tests for a documentation change (though, obviously, you'll have to wait to merge).

**Pull request merged**